plugins/azure_sentinel/plugin.spec.yaml

```diff
@@ -474,329 +483,1938 @@ types:
       description: Azure resource type
       type: string
   Incident:
-      id:
-        title: ID 
-        description: Full incident ID
-        type: string
-      name:
-        title: Name 
-        description: Incident name - short ID
-        type: string
-      type:
-        title: Type
-        description: Incident type
-        type: string
-      etag:
-        title: Etag
-        description: Etag
-        type: string
-      properties:
-        title: Incident Properties
-        description: Incident properties object
-        type: IncidentProperties
-connection:
-  client_id:
-    title: Client ID
-    description: The application ID that the application registration portal assigned
-      to your app
-    type: string
-    required: true
-    example: c163eff0-d1a1-4618-ee2a-453534f43cee
-  client_secret:
-    title: Client Secret
-    type: credential_secret_key
-    description: The application secret that you generated for your app in the app
-      registration portal
-    required: true
-    example: ef50c6bx9umaik9agvoxtoqec2fg9f0y
-  tenant_id:
-    title: Tenant ID
-    type: string
-    description: The Azure Tenant ID is a Global Unique Identifier (GUID) for your Azure Active Directory Tenant
-    required: true
-    example: 5ceea899-ae8c-4ff1-fffe-353646eeeff0
-actions:
-  create_update_incident:
-    title: Create or Update Incident
-    description: Creates a new incident or updates an existing incident 
-    input:
-      incidentId:
-        title: Incident ID
-        required: true
-        description: Incident ID
+    id:
+      title: ID
+      description: Full incident ID
+      type: string
+    name:
+      title: Name
+      description: Incident name - short ID
+      type: string
+    type:
+      title: Type
+      description: Incident type
+      type: string
+    etag:
+      title: Etag
+      description: Etag
```

Comment:
This is short for "Extended tag" is that correct? We may want to include the fully spelled out name in the description